pkg/operator/ceph/object/rgw.go

```diff
@@ -180,6 +182,9 @@ func (c *clusterConfig) startRGWPods(realmName, zoneGroupName, zoneName string,
 			return errors.Wrapf(err, "failed to set annotation for deployment %q", deployment.Name)
 		}
 
+		// apply cephx secret resource version to pod to ensure it restarts when keyring updates
+		deployment.Spec.Template.Annotations[keyring.CephxKeyIdentifierAnnotation] = secretResourceVersion
```

Comment:
Will this annotation be applied always, even if the secrets were never rotated?

Reply:
Oh, good point. It _could_ be optimized, but I'm not sure users gain a ton of benefit from going through the effort.